electricitylci/__init__.py

```diff
@@ -210,6 +210,40 @@ def write_generation_mix_database_to_dict(
     return genmix_dict
 
 
+def write_fuel_mix_database_to_dict(
+    genmix_database, gen_dict, regions=None
+):
+    from electricitylci.generation_mix import olcaschema_usaverage
+    if regions is None:
+        regions = config.model_specs.regional_aggregation
+    if regions in ["FERC","US","BA"]:
```

Comment:
it just dawned on me that you are trying to make changes here to bring in Canadian mixes for the eio trading method as well. I think for now the two processes should be segregated (see [this comment](https://github.com/USEPA/ElectricityLCI/pull/143/files#r471479953)), until we can do more changes to ensure that the Canadian mixes can generated when `replace_egrid` is true.

Reply:
The changes should only represent updates to the eGRID-level trading methodology. We were trying to address problems with missing providers and ensuring that international electricity traded in was represented using US avg generation by technology type. If there is some interaction with the other approach, that should be addressed.

Reply:
[See above comment.](https://github.com/USEPA/ElectricityLCI/pull/143#discussion_r471512114)

Reply:
> it just dawned on me that you are trying to make changes here to bring in Canadian mixes for the eio trading method as well. I think for now the two processes should be segregated (see [this comment](https://github.com/USEPA/ElectricityLCI/pull/143/files#r471479953)), until we can do more changes to ensure that the Canadian mixes can generated when `replace_egrid` is true.

Yeah, this was a mistake. I should have known that bringing in Canadian mixes for the eio trading method as well will not work with the current version of the code. The calculations are specific for the eGRID trading methodology only.

Reply:
Fixing the problem by moving the calls for us average mix dictionary creation and international mix creating withing the else: section of the main.py file which runs only when the epa_egrid_trading is False.
Not removing the lines from the __init__.py file as of now.

Reply:
@jump2conclusionsmatt  Is this solution ok?

Reply:
Yup, this should be fine. I'm just looking for some time to marge all the changes locally to run again before merging into master.